classes/fields/currency.php

```diff
@@ -327,8 +332,8 @@ public function pre_save( $value, $id = null, $name = null, $options = null, $fi
 	 */
 	public function format( $value = null, $name = null, $options = null, $pod = null, $id = null ) {
 
-		if ( null === $value ) {
-			// Don't enforce a default value here
+		if ( $this->is_empty( $value ) ) {
```

Comment:
`is_empty()` will catch `0` and `0.00` values too, not sure we want that applied here.